sounddevice.py

```diff
@@ -2265,10 +2265,66 @@ def reset(self):
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
-
+    # A simple enumeration of the Portaudio error codes
+    PaErrorCode = type('PaErrorCode', (), _ffi.typeof('PaErrorCode').relements)
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Parameters
+    ----------
+    msg : optional
+        User error message.
+    errcode : int or None, optional
```

Comment:
This should be `int, optional`.

Do we even need to document the constructor arguments?
Would a user really want to create a `PortAudioError`?